### PR TITLE
MMT-2254 updating proposal publish logging and adding comment to facilitate development on the manage proposals page

### PR DESCRIPTION
**Pull request by ryanmiller-1**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -126,6 +126,7 @@ def finish_successful_login(profile)
     # Stores additional information in the session pertaining to the user
     store_profile(profile)
     Rails.logger.debug "Successful URS Login by user #{authenticated_urs_uid}" if session[:login_method] == 'urs'
```
Comment (charleshuang80) [inline app/controllers/users_controller.rb:128]: is this not true for proposal mode?
If the new line is necessary, then I would combine it to the end of this line: `|| Rails.configuration.proposal_mode`

Comment (ryanmiller-1) [reply app/controllers/users_controller.rb:128]: Different logging levels. I assumed we don't have debug on in SIT+

Comment (charleshuang80) [reply app/controllers/users_controller.rb]: ok

**Comment by charleshuang80** [comment]
I'm good with the new changes also.

**Pull request merged**